CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# they will be requested for
+# review when someone opens a pull request.
+*       @JoeOster @anjgola @mkitez
+
+# This owner owns any files in the given
+# directory at the root of the repository and any of its
+# subdirectories.
+/DirectProgramming/DPC++/ @JoeOster @moushumi-maria, @pmpeter1
+/DirectProgramming/Jupyter/ @JoeOster @praveenkk123
+/DirectProgramming/C++/ @JoeOster @pmpeter1 @Propanu
+/DirectProgramming/Fortran/ @JoeOster @pmpeter1 @Propanu
+/Tools/ @JoeOster @Propanu
+/Libraries/ @JoeOster @Propanu @mav-intel @JoeOster
+
+# Reviwers for all sample.json modification 
+*sample.json @pfische1
```

Comment:
Can you please add myself here as well? I would like to be aware of the `ciTests` field changes in particular

Reply:
Ok